Makefile

```diff
@@ -1,16 +1,7 @@
 regenerate:
 	go install github.com/goccmack/gocc
-	gocc -zip -o ./internal/ dot.bnf 
+	gocc -zip -o ./internal/ dot.bnf
 	find . -type f -name '*.go' | xargs goimports -w
 
 test:
 	go test ./...
-
-travis:
```

Comment:
Is it possible to change this to `action` instead of `travis` and then call this from the github action.
That way it is possible to reproduce the github action locally before pushing?

Reply:
That's a really good idea, I forgot about the possibility to run locally... 

In order to have separated steps in github action, I will add different rules to Makefile (`dependencies`, `build` and `checkers`) and an `action` one that calls the others.